error[E0599]: no method named `count_ones` found for reference `&num_rug_adapter::Integer`

This is causing my Mac build to fail:
```
cargo build --no-default-features --features num --release
   Compiling scryer-prolog v0.8.128 (/Users/david/scryer-prolog)
error[E0599]: no method named `count_ones` found for reference `&num_rug_adapter::Integer` in the current scope
    --> src/machine/system_calls.rs:5546:67
     |
5546 |                     Ok(Number::Integer(n)) => Integer::from((&*n).count_ones().unwrap()),
     |                                                                   ^^^^^^^^^^ method not found in `&num_rug_adapter::Integer`

error: aborting due to previous error
```

--- a/crates/num-rug-adapter/src/lib.rs
+++ b/crates/num-rug-adapter/src/lib.rs
@@ -117,6 +117,10 @@ impl Integer {
     pub fn gcd(&self, other: &Self) -> Self {
         Integer(num_integer::Integer::gcd(&self.0, &other.0))
     }
+
+    pub fn count_ones(&self) -> Option<u32> {
+        Some(self.0.to_u32_digits().1.iter().map(|&d| d.count_ones()).sum())
+    }
 }
 
 impl From<&Integer> for Integer {

--- a/crates/num-rug-adapter/src/lib.rs
+++ b/crates/num-rug-adapter/src/lib.rs
@@ -118,6 +118,7 @@ impl Integer {
         Integer(num_integer::Integer::gcd(&self.0, &other.0))
     }
 
+    #[inline]
     pub fn count_ones(&self) -> Option<u32> {
         Some(self.0.to_u32_digits().1.iter().map(|&d| d.count_ones()).sum())
     }

